api/lib/stores/global-store.js

```diff
@@ -2,67 +2,50 @@ const Sentry = require('@sentry/node')
 
 class GlobalStore {
   constructor(database) {
-    this.stores = []
     this.db = database
+    this.store = {}
     this.globalValidators = {}
+    this.validatorsLookup = {}
 
     this.dataReady = new Promise((resolve) => {
       this.resolveReady = resolve
     })
     this.dataReady.then(() => {
       console.log(`Global Store is ready`)
     })
-    this.getStores()
+    this.getReady()
   }
 
-  async getStores() {
-    const networks = await this.db.getNetworks()
-    if (networks.length !== this.stores.length) {
-      setTimeout(async () => await this.getStores(), 1000)
+  async getReady() {
+    if (Object.keys(this.store).length === 0) {
+      setTimeout(async () => await this.getReady(), 1000)
```

Comment:
let'S avoid timeouts. why is this here?

Reply:
to give it some time to load until rechecking